plugins/hotkeyshortcuts.koplugin/defaults.lua

```diff
@@ -0,0 +1,117 @@
+-- typed on my Commodore 64 ;)
+local Device = require("device")
+
+return {
+    hotkeyshortcuts_fm = {
+        modifier_plus_up                 = nil,
+        modifier_plus_down               = nil, -- keep nil.
+        modifier_plus_left               = nil,
+        modifier_plus_right              = nil,
+        modifier_plus_left_page_back     = nil, -- keep nil.
+        modifier_plus_left_page_forward  = nil, -- keep nil.
+        modifier_plus_right_page_back    = nil, -- keep nil.
+        modifier_plus_right_page_forward = nil, -- keep nil.
+        modifier_plus_back               = {open_previous_document = true,},
+        modifier_plus_home               = Device:hasWifiToggle() and {toggle_wifi = true,} or {},
+        modifier_plus_press              = nil, -- keep nil.
+        modifier_plus_menu               = nil, -- keep nil.
+        -- optional
+        press = nil, -- keep nil
```

Comment:
What does "optional" means in this context ?
edit: reading more below, I guess it's dependant on `G_reader_settings:isTrue("press_key_does_hotkeyshortcuts")`, so you may write about that here too.